src/main/groovy/wooga/gradle/build/unity/UnityBuildPluginExtension.groovy

```diff
@@ -18,20 +18,19 @@
 package wooga.gradle.build.unity
 
 import org.gradle.api.file.ConfigurableFileCollection
-import org.gradle.api.file.Directory
 import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.FileCollection
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.provider.MapProperty
 import org.gradle.api.provider.Property
-import org.gradle.api.provider.Provider
 
 interface UnityBuildPluginExtension<T extends UnityBuildPluginExtension> {
 
     DirectoryProperty getAppConfigsDirectory()
     DirectoryProperty getOutputDirectoryBase()
     Property<String> getToolsVersion()
     Property<String> getVersion()
+    Property<UBSVersion> getUbsVersion()
```

Comment:
can we please call this `ubsCompatibilityVersion`. I don't want people to think that the version has to match with UBS. For me it means run the API in compatibility mode of version X. UBS will itself still support this. We can then also declare an API value in UBS to guard the kinds of changes. I wonder if we could even invoke unity with some form of compile define set ....

Reply:
Done